src/wp-includes/class-wp-customize-manager.php

```diff
@@ -2106,6 +2109,7 @@ public function remove_frameless_preview_messenger_channel() {
 		} )();
```

Comment:
I guess this is out of scope for this PR, but looks like we could prevent some other corruption here by using `URLSearchParams` instead of string-searching the query of the URL. no need for DOM either.

```js
url = new URL( location.href );
queryParams = url.searchParams;

if ( queryParams.has( 'customize_messenger_channel' ) ) {
	queryParams.delete( 'customize_messenger_channel' );
	url.search = queryParams;
	location.replace( url );
}

```

Reply:
Ah yes, this code dates back to a time before we could rely on `URL` or `URLSearchParams`. I made a similar change recently to `wp-embed` (r56383). Probably should be put into a new defect.

Reply:
Filed this in Core-59480